SwiftSampleProject/SwiftyVokoderTests/ManagedObjectContextTests.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ManagedObjectContextTests.swift
+//  SwiftSampleProject
+//
+//  Created by Carl Hill-Popper on 2/9/16.
+//  Copyright © 2016 Vokal.
+//
+
+import XCTest
+import Vokoder
+@testable import SwiftyVokoder
+
+class ManagedObjectContextTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        VOKCoreDataManager.sharedInstance().resetCoreData()
+        VOKCoreDataManager.sharedInstance().setResource("CoreDataModel", database: "CoreDataModel")
+        
+        Stop.vok_import(CTAData.allStopDictionaries())
+        VOKCoreDataManager.sharedInstance().saveMainContextAndWait()
+    }
+    
+    func testContextChain() {
+        let manager = VOKCoreDataManager.sharedInstance()
+
+        let tempContext = manager.temporaryContext()
+        //temp context is a child of the main context
+        XCTAssertEqual(tempContext.parentContext, manager.managedObjectContext)
+        //main context has a private parent context
+        XCTAssertNotNil(manager.managedObjectContext.parentContext)
+    }
+    
+    func testDeletingObjectsOnTempContextGetsSavedToMainContext() {
+        //get a temp context, delete from temp, save to main, verify deleted on main
+        let manager = VOKCoreDataManager.sharedInstance()
```

Comment:
Would it make sense to create properties out of `manager` and `tempContext` and do this configuration in `setUp`? - looks like it is pretty much the same for all tests